site/app/models/Button.php

```diff
@@ -46,6 +48,7 @@ public function __construct(Core $core, array $details) {
             $this->progress = floatval($this->progress);
         }
         $this->title_on_hover = $details["title_on_hover"] ?? false;
+        $this->aria_label = $details["aria_label"] ?? null;
```

Comment:
modify the tests in [ButtonTester](https://github.com/Submitty/Submitty/blob/master/site/tests/app/models/ButtonTester.php) to account for this change.